packages/suite/src/hooks/wallet/useBitcoinAmountUnit.ts

```diff
@@ -17,6 +25,9 @@ const UNIT_OPTIONS = [
 export const useBitcoinAmountUnit = () => {
     const bitcoinAmountUnit = useSelector(state => state.wallet.settings.bitcoinAmountUnit);
     const selectedAccount = useSelector(state => state.wallet.selectedAccount);
```

Comment:
Idea to simplify this hook and related components:

I think this doesn't have to depend on selected account, but on the network/symbol instead.  `useBitcoinAmountUnit` hook could have one parameter (networkSymbol) and that would simplify at least [AmountUnitSwitchWrapper.tsx](https://github.com/trezor/trezor-suite/pull/5526/files#diff-0206743f67db77e5f4e1533874085c76e43cc991c33b9578f2db4001c7c5ae60R45) where isActive param would be no more needed.

That networkSymbol to pass to this hook seems to be always available or obvious in places of use of this hook.

Reply:
done